pom.xml

```diff
@@ -205,57 +264,34 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>2.5.1</version>
         <configuration>
-          <encoding>UTF-8</encoding>
           <source>${java.target}</source>
           <target>${java.target}</target>
         </configuration>
       </plugin>
-      <!-- we need to create a uber-jar for https://github.com/ktoso/maven-git-commit-id-plugin/issues/196 -->
+
```

Comment:
If `maven.compiler.source` and `maven.compiler.target` would be set as properties to 1.7 we could remove the complete plugin configuration here.